src/main/java/org/llorllale/cactoos/matchers/MatcherOf.java

```diff
@@ -42,13 +43,14 @@
 /**
  * Func as Matcher.
  *
- * <p>There is no thread-safety guarantee.
+ * <p>
+ * There is no thread-safety guarantee.
  *
  * @param <T> Type of object to match
  * @since 0.12
- * @todo #135:30min Remove all constructors except the last one so that
- *  every matcher implemented using MatcherOf take care of properly
- *  describe itself and the mismatch.
+ * @todo #135:30min Remove all constructors except the last one so that every
+ *  matcher implemented using MatcherOf take care of properly describe itself
+ *  and the mismatch.
```

Comment:
@andreoss I'm not sure it's a good idea to modify existing todos, can you revert this and the other places where you modified them?

Reply:
@victornoel Done, was reformatted automatically